### PR TITLE
cocoapods and swift package manager support

### PR DESCRIPTION
**Pull request by biancajiang**
Experimental new analyzers to support SWIFT that we have used for a while internally:
- CocoaPodsAnalyzer.java
- SwiftPackageManagerAnalyzer.java
- handle when both co-exit in DependencyBundlingAnalyzer.java

### CODE REVIEW

**Comment by jeremylong** [comment]
@biancajiang Thank you for the PR! truly appreciate the enhancements and additions.

--Jeremy

**Comment by biancajiang** [comment]
@jeremylong this is an awesome project, better than some of the commercial solutions out there from our evaluations.  Happy to contribute!

Another PR coming on some Node package enhancements, just a heads up.

File: dependency-check-core/src/main/java/org/owasp/dependencycheck/analyzer/SwiftPackageManagerAnalyzer.java [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+    	String contents;
+        try {
+            contents = FileUtils.readFileToString(dependency.getActualFile(), Charset.defaultCharset());
+        } catch (IOException e) {
+            throw new AnalysisException(
+                    "Problem occurred while reading dependency file.", e);
+        }
+        final Matcher matcher = SPM_BLOCK_PATTERN.matcher(contents);
+        if (matcher.find()) {
+            contents = contents.substring(matcher.end());
```
Comment (jeremylong) [inline dependency-check-core/src/main/java/org/owasp/dependencycheck/analyzer/SwiftPackageManagerAnalyzer.java:134]: @biancajiang can you take a look at this line?  The variable 'contents' does not appear to be used.

Comment (biancajiang) [reply dependency-check-core/src/main/java/org/owasp/dependencycheck/analyzer/SwiftPackageManagerAnalyzer.java:134]: @jeremylong you are right, line 134 is unnecessary.  All of my test cases run successfully w/o it.

I just removed the branch for swift.  Pls let me know if you want me to create a PR for this.

Comment (jeremylong) [reply dependency-check-core/src/main/java/org/owasp/dependencycheck/analyzer/SwiftPackageManagerAnalyzer.java:134]: I removed line 134 - I just wanted confirmation that it was intentionally not used. Thanks!

**Pull request merged**